updates/1/updates.sql

```diff
@@ -26,3 +26,37 @@ INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xBBc2AE13b23d7
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xB1f66997A5760428D3a87D68b90BfE0aE64121cC", 18, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xB1f66997A5760428D3a87D68b90BfE0aE64121cC", "C", "LuaSwap", "LUA", 1601024188, NULL, "lua", "LUA", "0xB1f66997A5760428D3a87D68b90BfE0aE64121cC");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x08d967bb0134F2d07f7cfb6E246680c53927DD30", 18, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x08d967bb0134F2d07f7cfb6E246680c53927DD30", "C", "MATH", "MATH", 1569577860, NULL, "math", "MATH", "0x08d967bb0134F2d07f7cfb6E246680c53927DD30");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x965697b4ef02F0DE01384D0d4F9F782B1670c163", 6, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x965697b4ef02F0DE01384D0d4F9F782B1670c163", "C", "Oxygen", "OXY", 1607784270, NULL, "oxygen", "OXY", "0x965697b4ef02F0DE01384D0d4F9F782B1670c163");
+*
+UPDATE assets SET swapped_for="_ceth_0x0FD10b9899882a6f2fcb5c371E17e70FdEe00C38" WHERE identifier="_ceth_0xA15C7Ebe1f07CaF6bFF097D8a589fb8AC49Ae5B3";/*Introduce NPXS token swap*/
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xA15C7Ebe1f07CaF6bFF097D8a589fb8AC49Ae5B3", 18, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xA15C7Ebe1f07CaF6bFF097D8a589fb8AC49Ae5B3", "C", "NPXS", "NPXS", 1520494140, NULL, "pundi-x-old", "NPXS", "0xA15C7Ebe1f07CaF6bFF097D8a589fb8AC49Ae5B3");
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x0FD10b9899882a6f2fcb5c371E17e70FdEe00C38", 18, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x0FD10b9899882a6f2fcb5c371E17e70FdEe00C38", "C", "Pundi X", "PUNDIX", 1615175880, NULL, "pundi-x-new", "PUNDIX", "0x0FD10b9899882a6f2fcb5c371E17e70FdEe00C38");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x33D0568941C0C64ff7e0FB4fbA0B11BD37deEd9f", 6, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x33D0568941C0C64ff7e0FB4fbA0B11BD37deEd9f", "C", "RAMP", "RAMP", 1597966623, NULL, "ramp", "RAMP", "0x33D0568941C0C64ff7e0FB4fbA0B11BD37deEd9f");
```

Comment:
As per https://etherscan.io/address/0x33d0568941c0c64ff7e0fb4fba0b11bd37deed9f#readContract:

name should be `RAMP DEFI`

```suggestion
INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x33D0568941C0C64ff7e0FB4fbA0B11BD37deEd9f", 6, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x33D0568941C0C64ff7e0FB4fbA0B11BD37deEd9f", "C", "RAMP DEFI", "RAMP", 1597966623, NULL, "ramp", "RAMP", "0x33D0568941C0C64ff7e0FB4fbA0B11BD37deEd9f");
```